frontend/Props.js

```diff
@@ -16,10 +16,7 @@ var PropVal = require('./PropVal');
 class Props extends React.Component {
   props: Object;
   shouldComponentUpdate(nextProps: Object): boolean {
-    if (nextProps === this.props) {
-      return false;
-    }
-    return true;
+    return nextProps !== this.props;
```

Comment:
This was actually a pretty weird comparison. I wonder if @jaredly meant to compare `this.props.props` with `nextProps.props` instead.